pkg/controller/seed-controller-manager/kubernetes/pending.go

```diff
@@ -124,7 +124,12 @@ func (r *Reconciler) ensureClusterNetworkDefaults(ctx context.Context, cluster *
 
 	if cluster.Spec.ClusterNetwork.ProxyMode == "" {
 		setProxyMode := func(c *kubermaticv1.Cluster) {
-			c.Spec.ClusterNetwork.ProxyMode = resources.IPVSProxyMode
+			// IPVS causes issues with Hetzner's LoadBalancers
+			if c.Spec.Cloud.Hetzner != nil {
+				c.Spec.ClusterNetwork.ProxyMode = resources.IPTablesProxyMode
+			} else {
+				c.Spec.ClusterNetwork.ProxyMode = resources.IPVSProxyMode
+			}
```

Comment:
Without this, LBs never become healthy, even though the nodeport is open. See

* https://github.com/hetznercloud/hcloud-cloud-controller-manager/issues/58#issuecomment-654347439
* https://github.com/hetznercloud/hcloud-cloud-controller-manager/issues/58#issuecomment-659170459
* https://github.com/metallb/metallb/issues/153